pom.xml

```diff
@@ -120,6 +132,16 @@
             <artifactId>jaxb-api</artifactId>
             <version>2.3.0</version>
         </dependency>
+        <dependency>

```

Comment:
Do we really need the following two dependencies? I have run the application without them and works well.

Reply:
On a windows environment, the maven package goal doesn't include theses dependencies on the final jar. So, at runtime, we have a ClassNotFoundException. To solve this, I have had to add it explicitly. 
But, we should find a way to guarantee that the environment doesn't affect in the maven lifecycle. Does anyone else have this problem under a Windows environment?
I can remove these dependencies from pom file and push it again without it.
Thx

Reply:
Thank you @jazzinjars. I understand the reason. The for me is fine. Great PR